css/style.css

```diff
@@ -0,0 +1,79 @@
+body{
+  background-color: #f3f4f5;
+  padding: 15px;
+  font-family: Helvetica;
+  font-size: 14px;
+  background-image: url(http://www.gannett-cdn.com/-mm-/d8b68461f8c54f619579b4b1091145ba082f9ab2/c=0-1287-4000-3547&r=x1683&c=3200x1680/local/-/media/2016/05/08/USATODAY/USATODAY/635983284547965765-Radio-AMoon-Cover-240416.jpg);
+  background-repeat: no-repeat;
+  background-position: absolute;
+  background-size: auto;
+  background-attachment: fixed;
+  opacity: 0.9;
+  width: auto;
+  height: auto;
+}
+nav{
+  font-size: 20px;
+  text-align: center;
+  color: #808080;
+  padding: 5px;
+  margin: 5px;
+  font-family: sans-serif;
+}
+h1{
+  background-color: white;
+  text-align: center;
+  font-family: sans-serif;
+  padding: 20px;
+  margin: 20px;
+  letter-spacing: 3px;
+}
+img{
+  width: 600;
+  height: auto;
+  float: right;
+  vertical-align: top;
+
+}
+div{
+  align: right;
+  justify-content: center;
```

Comment:
@jcooper8190 `justify-content: center;` is used to center items when using flexbox. It cannot be used outside of a flex container.